compute_field_after_install/models/recompute_field.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+import logging
+
+from openerp import api, fields, models
+from openerp.exceptions import Warning as UserError
+from openerp.tools.translate import _
+
+from odoo.tools import config
+
+_logger = logging.getLogger(__name__)
+
+
+class RecomputeField(models.Model):
+    _name = "recompute.field"
+    _description = "Recompute Field"
+
+    model = fields.Char(required=True)
+    field = fields.Char(required=True)
+    last_id = fields.Integer(
+        string="Last ID", help="Last record ID on which computing have been executed"
+    )
+    step = fields.Integer(required=True, default=1000, help="Recomputing batch size.")
+    state = fields.Selection(
+        [
+            ("todo", "Todo"),
+            ("done", "Done"),
+        ]
+    )
+
+    @api.model
+    def _run_all(self):
+        return self.search([("state", "=", "todo")]).run()
```

Comment:
If this compute took too long on module installation, it will also take too long in a cron job, unless it's run in batches. Can you support a batch_size=SOME_DEFAULT parameter? People can then adjust this in the scheduled action, before they enable it.